README.md

```diff
@@ -10,4 +10,6 @@ This is a template repository for running [Wordpress](https://wordpress.org) on
 
 ## Deployment
 
+[![Deploy to Render](https://render.com/images/deploy-to-render-button.svg)](https://render.com/deploy)
```

Comment:
I think we're trying to add the reverse link into the deploy link query param, since some browser cases block us from reading the referrer from the deploy page.

Reply:
This makes sense and I will do so.